documentation/modules/configuring/proc-config-kafka-connect.adoc

```diff
@@ -13,17 +13,18 @@ The example shown in this procedure is for the `KafkaConnect` resource, but the
 .Kafka connector configuration
 KafkaConnector resources allow you to create and manage connector instances for Kafka Connect in a Kubernetes-native way.
 
-In the configuration, you enable KafkaConnectors for a Kafka Connect cluster by adding the `strimzi.io/use-connector-resources` annotation.
-You can also specify external configuration for Kafka Connect connectors through the `externalConfiguration` property.
+In your Kafka Connect configuration, you enable KafkaConnectors for a Kafka Connect cluster by adding the `strimzi.io/use-connector-resources` annotation.
+You can also add `build` configuration so that Strimzi automatically builds a container image with the connector plugins you require for your data connections.
```

Comment:
```suggestion
You can also add a `build` configuration so that Strimzi automatically builds a container image with the connector plugins you require for your data connections.
```